resources/views/account/partials/orders.blade.php

```diff
@@ -1,20 +1,45 @@
-<template v-if="data.customer.orders.items.length">
-    <x-rapidez-ct::sections>
-        <x-rapidez-ct::card.inactive class="flex-col" v-for="order in data.customer.orders.items">
-            <x-rapidez-ct::title.lg>
-                @lang('Order') #@{{ order.number }}
-            </x-rapidez-ct::title.lg>
-            <a :href="'/account/order/'+order.number">
-
+<div>
```

Comment:
is this <div neccesary?

Reply:
The reason for this is that the wrapper is a template, which can only contain one element as its direct child. Otherwise, JavaScript errors would occur. The wrapper div prevents this by enclosing the template content in a single element.